views/event-detail.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layout.html" %}
+
+{% block mainContent %}
+  <div>
+    <p>What: {{event.title}}</p>
+    <p>When: {{event.date}}</p>
```

Comment:
Not sure I'm using the most elegant form, but I think I have this fixed --
look for it in my next pull request. Here's the code I used:

<p>When: <time
datetime="{{event.date}}">{{event.date|prettyDate}}</time></p>

On Mon, Feb 9, 2015 at 10:23 PM, David Hatch <notifications@github.com>
wrote:

> In views/event-detail.html
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mightyLake_mgt656-2Dspring-2D2015-2Dproject_pull_25-23discussion-5Fr24386706&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=qcJxOJmIpoQX59UDHnw6vYS8QNZlleLqZuSHaMKiP_4&m=-ftZhlmOCzr1amuAymlDsWMGc6AJPFojTFzg4A1y8eI&s=AL_HRkT00G4zZd1NG7_8jsch5JdHT0OTUx1gLeTzwPM&e=>
> :
>
> > @@ -0,0 +1,34 @@
> > +{% extends "layout.html" %}
> > +
> > +{% block mainContent %}
> > +  <div>
> > +    <p>What: {{event.title}}</p>
> > +    <p>When: {{event.date}}</p>
>
> This date should be formatted better, using the time tag, setting the
> datetime attribute, and including a formatted date using javascript (e.g.
> Feb 20, 2015, or something like that).
>
> —
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mightyLake_mgt656-2Dspring-2D2015-2Dproject_pull_25_files-23r24386706&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=qcJxOJmIpoQX59UDHnw6vYS8QNZlleLqZuSHaMKiP_4&m=-ftZhlmOCzr1amuAymlDsWMGc6AJPFojTFzg4A1y8eI&s=QLhLg4RfjmYYGP_3cTw-o7d2GB5P3ehjNQotFajdtLA&e=>
> .
>